apiserver/common/firewall/state.go

```diff
@@ -29,12 +29,13 @@ type State interface {
 }
 
 // TODO(wallyworld) - for tests, remove when remaining firewaller tests become unit tests.
-func StateShim(st *state.State) stateShim {
-	return stateShim{st}
+func StateShim(st *state.State, m *state.Model) stateShim {
```

Comment:
Is model really needed here? Why not just call st.Model() where a model is required?